schemas/external/ui/support-form/support-form.json

```diff
@@ -0,0 +1,33 @@
+{
+    "//playground": "https://rjsf-team.github.io/react-jsonschema-form/#eyJmb3JtRGF0YSI6eyJhcHBsaWNhdGlvblR5cGUiOiJIZWxtIENoYXJ0In0sInNjaGVtYSI6eyJ0aXRsZSI6IlVwbG9hZCBBcHBsaWNhdGlvbiBGaWxlIiwicHJvcGVydGllcyI6eyJuYW1lIjp7InR5cGUiOiJzdHJpbmciLCJ0aXRsZSI6IkdpdmUgeW91ciBhcHBsaWNhdGlvbiBmaWxlIGEgbmFtZSIsIngtcmpzZi1ncmlkLWFyZWEiOiI2In0sImFwcGxpY2F0aW9uVHlwZSI6eyJ0aXRsZSI6IlNlbGVjdCBhcHBsaWNhdGlvbiB0eXBlIiwiZW51bSI6WyJIZWxtIENoYXJ0IiwiS3ViZXJuZXRlcyBNYW5pZmVzdCIsIkRvY2tlciBDb21wb3NlIl0sImRlZmF1bHQiOiJIZWxtIENoYXJ0IiwieC1yanNmLWdyaWQtYXJlYSI6IjYifX0sImRlcGVuZGVuY2llcyI6eyJhcHBsaWNhdGlvblR5cGUiOnsib25lT2YiOlt7InByb3BlcnRpZXMiOnsiYXBwbGljYXRpb25UeXBlIjp7ImNvbnN0IjoiSGVsbSBDaGFydCJ9LCJ1cmwiOnsidHlwZSI6InN0cmluZyIsImZvcm1hdCI6InVyaSIsInRpdGxlIjoiVVJMIiwiZGVzY3JpcHRpb24iOiJyYXcgVVJMIG9mIHRoZSBhcHBsaWNhdGlvbiBmaWxlLCBleGFtcGxlOiBodHRwczovL3Jhdy5naXRodWIuY29tLy4uLiIsIngtcmpzZi1ncmlkLWFyZWEiOiIxMiJ9fSwicmVxdWlyZWQiOlsidXJsIl19LHsicHJvcGVydGllcyI6eyJhcHBsaWNhdGlvblR5cGUiOnsiY29uc3QiOiJLdWJlcm5ldGVzIE1hbmlmZXN0In19LCJvbmVPZiI6W3sidGl0bGUiOiJVUkwgVXBsb2FkIiwicHJvcGVydGllcyI6eyJ1cmwiOnsidHlwZSI6InN0cmluZyIsImZvcm1hdCI6InVyaSIsInRpdGxlIjoiVVJMIiwiZGVzY3JpcHRpb24iOiJyYXcgVVJMIG9mIHRoZSBhcHBsaWNhdGlvbiBmaWxlLCBleGFtcGxlOiBodHRwczovL3Jhdy5naXRodWIuY29tLy4uLiIsIngtcmpzZi1ncmlkLWFyZWEiOiIxMiJ9fSwicmVxdWlyZWQiOlsidXJsIl19LHsidGl0bGUiOiJGaWxlIFVwbG9hZCIsInByb3BlcnRpZXMiOnsiZmlsZSI6eyJ0eXBlIjoic3RyaW5nIiwiZm9ybWF0IjoiZGF0YS11cmwiLCJ0aXRsZSI6IkZpbGUgVXBsb2FkIiwiZGVzY3JpcHRpb24iOiJCcm93c2UgdGhlIGFwcGxpY2F0aW9uIGZpbGUgZnJvbSB5b3VyIGZpbGUgc3lzdGVtIiwieC1yanNmLWdyaWQtYXJlYSI6IjEyIn19LCJyZXF1aXJlZCI6WyJmaWxlIl19XX0seyJwcm9wZXJ0aWVzIjp7ImFwcGxpY2F0aW9uVHlwZSI6eyJjb25zdCI6IkRvY2tlciBDb21wb3NlIn19LCJvbmVPZiI6W3sicHJvcGVydGllcyI6eyJ1cmwiOnsidHlwZSI6InN0cmluZyIsImZvcm1hdCI6InVyaSIsInRpdGxlIjoiVVJMIiwiZGVzY3JpcHRpb24iOiJyYXcgVVJMIG9mIHRoZSBhcHBsaWNhdGlvbiBmaWxlLCBleGFtcGxlOiBodHRwczovL3Jhdy5naXRodWIuY29tLy4uLiIsIngtcmpzZi1ncmlkLWFyZWEiOiIxMiJ9fSwicmVxdWlyZWQiOlsidXJsIl19LHsicHJvcGVydGllcyI6eyJmaWxlIjp7InR5cGUiOiJzdHJpbmciLCJmb3JtYXQiOiJkYXRhLXVybCIsInRpdGxlIjoiRmlsZSBVcGxvYWQiLCJkZXNjcmlwdGlvbiI6IkJyb3dzZSB0aGUgYXBwbGljYXRpb24gZmlsZSBmcm9tIHlvdXIgZmlsZSBzeXN0ZW0iLCJ4LXJqc2YtZ3JpZC1hcmVhIjoiMTIifX0sInJlcXVpcmVkIjpbImZpbGUiXX1dfV19fX0sInVpU2NoZW1hIjp7ImFwcGxpY2F0aW9uVHlwZSI6eyJ1aTp3aWRnZXQiOiJyYWRpbyJ9fSwidGhlbWUiOiJtYXRlcmlhbC11aS01IiwibGl2ZVNldHRpbmdzIjp7fX0=",
+    "title": "Support Form",
+    "properties": {
+        "subject": {
+            "type": "string",
+            "title": "Subject",
+            "minLength": 1,
+            "x-rjsf-grid-area": "12"
```

Comment:
please add `description` field here with description of each field, try to make sure it is not short and contain all necessary information for reference you can check other schemas